gap/SkeletalFinSetsForCAP.gd

```diff
@@ -42,6 +52,19 @@ DeclareAttribute( "AsList",
 
 #! @Section Skeletal Constructors
 
+#! @Description
+#!  Construct a category of skeletal finite sets.
+#! @Returns a &CAP; category
+DeclareOperation( "CategoryOfSkeletalFinSets", [] );
```

Comment:
Should this be an operation or simply a global function?

Reply:
I don't know if there might be versions of higher arity of this operation in the future.

You decide.

Reply:
I have talked to Max and he has no concerns about a 0-ary operation, so we can leave it as it is.